### PR TITLE
Cypress improvements

### PR DESCRIPTION
**Pull request by mroz22**
regular cypress session output:
- b31c027 updated cypress to the lastest version
- c2fce8c re-enabled test 
- cd0b2b7 removed unused script, not really related to cypress
-  a3915f2 unified waiting for trezor-user-env in suite-web tests with other packages
- 9904cff renaming for better readability (group vs stage)
- 9fba2b5 refactored guide tests to be more future-proof
- 9d0a532 snapshot jobs available sooner. no need to wait for regular tests to finish.

I think we don't keep these commits, they may be squashed :thinking:

### CODE REVIEW

**Comment by Nodonisko** [comment]
Could you enable linting of JS files for this package? I think currently it is looking only for TS files.

**Comment by mroz22** [comment]
This looks like a problem that touches all packages. So far I have only added eslintrc.js file in 89d9021 just to make it consistent with other packages. I suggest that we should enable linting for js files globally.

**Comment by Nodonisko** [comment]
@mroz22 Definitely agree, I already have this on code improvements todo. So lets solve this issue separately.

File: yarn.lock [hunk 1]
```diff
@@ -7664,7 +7664,7 @@ buffer@^4.3.0:
     ieee754 "^1.1.4"
     isarray "^1.0.0"
 
-buffer@^5.1.0:
+buffer@^5.1.0, buffer@^5.6.0:
```
Comment (tomasklim) [inline yarn.lock]: Do not forget to run dedupe

File: packages/integration-tests/projects/suite-web/run_tests.js [hunk 2]
```diff
@@ -61,23 +59,6 @@ const wait = (timeout) => {
 }
 
 async function runTests() {
```
Comment (tomasklim) [inline packages/integration-tests/projects/suite-web/run_tests.js]: Nit: all functions except of `runTests` and `getTestFiles` have arrow functions

File: packages/integration-tests/projects/suite-web/tests/suite/guide.test.ts [hunk 3]
```diff
@@ -3,52 +3,65 @@
...
+            .children()
+            .first()
+            .then(el => {
+                const text = el.text();
+                el.click();
+                cy.log('text', text);
+                cy.getTestElement('@guide/label').should('have.text', text);
+            });
+        cy.getTestElement('@guide/nodes').first().children().first().click();
+        cy.getTestElement('@guide/headerBreadcrumb/categoryLink').click();
```
Comment (tomasklim) [inline packages/integration-tests/projects/suite-web/tests/suite/guide.test.ts]: nit: the only `data-test` which is in camelCase

File: packages/integration-tests/projects/suite-web/tests/suite/guide.test.ts [hunk 4]
```diff
@@ -3,52 +3,65 @@
...
+
+        // Search input
+        cy.getTestElement('@guide/search').type('trezor');
+        cy.getTestElement('@guide/search/results').children().should('have.length.above', 0);
+        cy.getTestElement('@guide/search').type('meow-wuf-nonsense');
+        cy.getTestElement('@guide/search/no-results');
     });
 
-    it('fills feedback form', () => {
+    it('in onboarding with device', () => {
```
Comment (tomasklim) [inline packages/integration-tests/projects/suite-web/tests/suite/guide.test.ts:55]: note: we should start using proper unified naming

File: packages/integration-tests/.eslintrc.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (tomasklim) [inline packages/integration-tests/.eslintrc.js]: Maybe there will be some collision with #4794

**Review by tomasklim — CHANGES_REQUESTED** [review]
Looks good. Just some nitpicks and notes

File: packages/integration-tests/.eslintrc.js (hunk 5 above)
Comment (mroz22) [reply packages/integration-tests/.eslintrc.js]: file removed

File: packages/integration-tests/projects/suite-web/tests/suite/guide.test.ts (hunk 4 above)
Comment (mroz22) [reply packages/integration-tests/projects/suite-web/tests/suite/guide.test.ts:55]: ok :D I am not sure what is the convention

File: packages/integration-tests/projects/suite-web/tests/suite/guide.test.ts (hunk 3 above)
Comment (mroz22) [reply packages/integration-tests/projects/suite-web/tests/suite/guide.test.ts]: changed

File: packages/integration-tests/projects/suite-web/run_tests.js (hunk 2 above)
Comment (mroz22) [reply packages/integration-tests/projects/suite-web/run_tests.js]: changed

File: yarn.lock (hunk 1 above)
Comment (mroz22) [reply yarn.lock]: done

**Review by tomasklim — APPROVED** [review]

**Pull request merged**